db/migrations/20200601071442-test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
I might be easier to ignore any generated js in terms of linting and formatting. Can we not an exclude for ```db/*```?

Reply:
Good point 👍